### PR TITLE
ShA/add breadcrumbs for topics

### PR DESCRIPTION
**Pull request by AlekseiiShevchuk**
I did it with the expectation that in the near future category slug will be implemented (i use slug in category link, but at the moment it does not work correctly)

### CODE REVIEW

File: resources/assets/js/app/views/topics/topicLayout.js [hunk 1]
```diff
@@ -4,17 +4,29 @@ var app = require('../../instances/appInstance');
...
     template: 'topicLayout',
     regions: {
-        container: '#posts'
+        container: '#posts',
+        breadcrumbs: '#categoryForBreadcrumbs'
+    },
+    events: {
+        'change': function () {
+            this.render();
+        }
```
Comment (binaryvo) [inline resources/assets/js/app/views/topics/topicLayout.js:30]: You can use short form instead:

 events: {
        'change': 'render'
...

File: resources/assets/templates/topic/topicCategoryForBreadcrumbs.hbs [hunk 2]
```diff
@@ -0,0 +1 @@
+Current category: <a href="#categories/{{slug}}/topics">{{ name }}</a>
```
Comment (binaryvo) [inline resources/assets/templates/topic/topicCategoryForBreadcrumbs.hbs]: Please remove words 'Current category:'

File: resources/assets/templates/topic/topicDetail.hbs [hunk 3]
```diff
@@ -1,4 +1,8 @@
+
 <div class="topic">
+    <span>
+        Current category: <a href="#categories/{{model.category.slug}}/topics">{{ model.category.name }}</a>
+    </span>
```
Comment (binaryvo) [inline resources/assets/templates/topic/topicDetail.hbs]: Please move this block inside of topic bloc, according to mocup: below topic description, above tags

**Comment by binaryvo** [comment]
Please fix remarks, update branch from next and reopen PR

**Comment by AlekseiiShevchuk** [comment]
All done

File: resources/assets/templates/topic/topicDetail.hbs [hunk 4]
```diff
@@ -9,13 +9,17 @@
...
             <div class="topic-body">
                 <div class="topic-header">
                     <h4>{{ model.name }}</h4>
                 </div>
                 <div class="topic-msg">
                     <p> {{{ model.generated_description }}}</p>
                 </div>
+                <span>
+                <a href="#categories/{{model.category.slug}}/topics">{{ model.category.name }}</a>
+                </span>
```
Comment (binaryvo) [inline resources/assets/templates/topic/topicDetail.hbs:22]: Please add 'Category: ' before category name

**Comment by AlekseiiShevchuk** [comment]
Done

File: resources/assets/templates/topic/topicCategoryForBreadcrumbs.hbs [hunk 5]
```diff
@@ -0,0 +1 @@
+<a href="#categories/{{slug}}/topics">{{ name }}</a>
```
Comment (binaryvo) [inline resources/assets/templates/topic/topicCategoryForBreadcrumbs.hbs]: Sorry, but seems like we don't need link here. Just text. But it should be like header text

**Comment by binaryvo** [comment]
Please make small change and re-open PR

**Comment by AlekseiiShevchuk** [comment]
Done

**Pull request merged**